tests/test_translations.h

```diff
@@ -0,0 +1,75 @@
+/*  test_translations.h                                                  */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2020 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2020 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#ifndef TEST_TRANSLATIONS_H
+#define TEST_TRANSLATIONS_H
+
+#include "core/io/resource_loader.h"
+#include "core/os/memory.h"
+#include "core/string/translation.h"
+#include "editor/import/resource_importer_csv_translation.h"
+
+namespace TestTranslations {
+
+TEST_CASE("[Translation] CSV import") {
+	TranslationServer *translation_server = nullptr;
+	translation_server = memnew(TranslationServer);
+	translation_server->setup(); //register translations, load them, etc.
+	translation_server->load_translations();
+
+	Ref<ResourceImporterCSVTranslation> import_csv_translation;
+	import_csv_translation.instance();
+
+	Map<StringName, Variant> options;
+	options["compress"] = false;
+	options["delimiter"] = 0;
+
+	List<String> gen_files;
+
+	Error result = import_csv_translation->import("tests/data/translations.csv", "", options,
```

Comment:
The file path should be relative to the binary, so that tests pass regardless of the folder you run the Godot binary from.

See https://github.com/godotengine/godot/blob/master/tests/test_pck_packer.h#L82-L86 for an example of doing this.

Reply:
Ok, I just copied path from `tests/test_file_access.h`. I'll change it.

Reply:
My bad, should be fixed there as well eventually.